source/main_cr.f90

```diff
@@ -52,6 +52,9 @@ program maincr
 #ifdef CR
   use checkpoint_restart
 #endif
+#ifdef BOINC
+  use mod_boinc
+#endif
```

Comment:
`only` is only really needed in module headers, to avoid leaking, or if you have variable name conflicts. All variables in `mod_boinc` are private as well.